.github/workflows/migrations.yml

```diff
@@ -33,6 +33,31 @@ jobs:
           token: ${{ github.token }}
           filters: .github/file-filters.yml
 
+  check-migration-approval:
+    name: check if migration is approved
+    runs-on: ubuntu-22.04
+    timeout-minutes: 3
+    needs: did-migration-change
+    if: needs.did-migration-change.outputs.added == 'true'
+    steps:
+      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # v3.1.0
+        with:
+          persist-credentials: false
```

Comment:
this won't work.  it needs to re-trigger on approval somehow -- this only runs on push

Reply:
Yep, I was just using this as a means of testing. I am planning on moving this to a separate workflow 👍🏻